File: II course/Programming Lab/Figures/Figure_Classes_1/IPolyPoint.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Figure_Classes_1
{
    public interface IPolyPoint
    {
        Point2D getP(int i); // возвращает координаты i-ой вершины многоугольника в виде объекта Point2D
        void setP(Point2D p, int i); // устанавливает новые координаты i-ой вершины многоугольника на основе объекта Point2D
    }
}

File: III course/Java programming/FiguresFX/src/main/java/com/figures/interfaces/IPolyPoint.java
package com.figures.interfaces;

import com.figures.figures.points.Point2D;

public interface IPolyPoint {
    Point2D getP(int i); // возвращает координаты i-ой вершины многоугольника в виде объекта Classes.Point2D
    void setP(Point2D p, int i); // устанавливает новые координаты i-ой вершины многоугольника на основе объекта Classes.Point2D
}
